### ISSUE TITLE
(WCRP) Encourage the use of qualifications/multiple registrations for major championships 

### ISSUE THREAD

**Issue by Samuel-Baird** [thewca/wca-documents#368]
`9.5.1 Organization teams should facilitate the registration process for the competitors in the region who have a high likelihood of winning and securing championship titles. This can be done by using qualification results and holding multiple registration periods (See 5,1 6.2.3)`

**Comment by EdHollingdale** [comment]
I don't think the WCRP is the correct location for this recommendation. There are many suggestions and recommendations which apply to major championships, and a separate document is likely required. I also do not feel that a policy is the best place for recommendations.

**Comment by nsilvestri** [comment]
Why not make it a requirement? That doesn't seem unreasonable to me for championships. And in the case where such a requirement is not anticipated to be necessary, it at least is not harmful.

**Comment by dmint789** [comment]
I agree this should be a requirement and that this is a suitable place for it

**Comment by EdHollingdale** [comment]
1) I think it could be incredibly harmful if not necessary - many major championships have budgets with high degrees of risk which require as many competitors to sign up as possible. Here the concern isn't that registration fills too quickly, but that it doesn't come close to filling. In these cases having a complex system of registration waves that is not clear to all but the most experienced competitor, and having it so that regular competitors have less time to sign up would create additional financial risk.
2) I think there is scope for a whole separate guidelines document for major championships to capture best practice. That should include comments on managing registrations, as well as things such as not scheduling semi-finals simultaneously. I have drafted such a document and it will continue to be refined I'm sure. 
3) However, I think major championship points should be guidelines initially and not additional requirements. I personally think holding major championships to higher standards than regular competitions makes complete sense and reflects the higher expectations that exist for them already. However, whenever I've mentioned ideas like having "tiers" to competitions and different rules/requirements that apply to them it has been very controversial, so I'd suggest we first have guidelines and then *think about* making some things mandatory at a later stage.

**Comment by viroulep** [comment]
If the problem is to make sure top cubers from [region] are registered for an event that crowns the [region] champion, then there are more than one way to do that, and making just one of the possibility a requirement is too narrow-minded in my opinion.
A few additional options to the registration periods:
  - simply restrict the competition to the people from [region].
  - picking a few events that allow only the top X (where X is intentionally low), to free up space in the schedule to accommodate more people.
  - achieve the same goal as the previous point, but by taking out one or more time consuming event from the competition (eg: having one FMC [Region] Championship in addition to the [Region] Championship).
  - create an unofficial invitational event where only the top 8/12/X are invited to be officially crowned the [region] champion (which would also let organizers/the WCA make some changes to the format), and keep a "[Regional] Convention" targetted at the international community.

I want to make clear that these options are not necessarily what I would like to see, I actually included option(s) in there that I would really not like to see.
My point is that reacting to "[Insertanamehere] is not registered" by making a very specific "solution" a requirements is not a good idea in my opinion.
I'd rather much have a proper document explaining what are the WCA expectations from a major championship, the various parameters to balance, what has been (un)successfully tried, and potential room for adjustments when things don't go as expected (this goes both ways, for instance when you get only 500ish competitors out of 1200 for a competition in a city easily reachable from anywhere in Europe, or when you get 1200 out of 1000 for a competition in a city linked to no other countries by plane).

**Comment by nsilvestri** [comment]
I think that there's space for both a document that helps championship organizers put on a successful event and a policy that helps championships be championships. A policy that requires championship organizers to guarantee top-ranked competitors the opportunity to register doesn't need to specify a mechanism by which to do that. Modifying Sam's initial proposal, I'm thinking something like:

> 9.5.1 Organization teams must facilitate the registration process for the competitors in the region who have a high likelihood of winning and securing championship titles.

Now there's nothing regulating what "facilitate the registration process" nor what "high likelihood of winning and securing championship titles" looks like, allowing organizers to choose the mechanisms that work best for a particularly region and championship.

**Comment by slongbehn** [comment]
The WCAT, alongside the Board, determined this is far too logistically challenging. The WRC allowed restricted advancements [9p2b](https://www.worldcubeassociation.org/regulations/full/#9p2b) which addresses a different way in spirit in #1263. This can close.
